### PR TITLE
 Marketplace suggestions in Edit Product screen

### PR DESCRIPTION
**Pull request by kovshenin**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Adds a new tab to Woo's product data metabox and renders the appropriate containers for JS to inject some Marketplace suggestions.

### How to test the changes in this Pull Request:

1. Checkout this branch and login as somebody with `install_plugins` caps
2. Create a new product or edit an existing product
3. Open the Extensions tab in the Product data metabox

### Screenshot
screen shot 2019-02-25 at 4 28 15 pm

### CODE REVIEW

**Review by haszari — COMMENTED** [review]
I've fixed up the CSS and added logic for hiding the tab/content when there are no suggestions to display.

Have tested this and is looking and working well :)

File: includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php [hunk 1]
```diff
@@ -24,13 +24,44 @@ public static function init() {
...
+	 *
+	 * Adds a new Extensions tab to the product data meta box.
+	 *
+	 * @param array $tabs Existing tabs.
+	 *
+	 * @return array
+	 */
+	public static function product_data_tabs( $tabs ) {
+		$tabs['marketplace-suggestions'] = array(
+			'label'    => _x( 'Get more options', 'Marketplace suggestions', 'woocommerce' ),
```
Comment (mikejolley) [inline includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php]: See more options? Not keen on this copy :)

**Review by mikejolley — COMMENTED** [review]
Didn't work for me I assume because of data, but the code looks ok. Not keen on 1 string :)

**Comment by kovshenin** [comment]
> I assume because of data

What do you mean? The data should be good to go in the .json file it's currently pointing to.

**Comment by mikejolley** [comment]
@kovshenin In that case the panel did not appear for me. Does this branch work solo or are there extra steps I'm missing?

**Comment by kovshenin** [comment]
@mikejolley if you have a fresh cached copy of the .json file it should work without any extra steps (unless you dismissed everything earlier).

**Comment by mikejolley** [comment]
Flushing transients did it thanks.

File: includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php (hunk 1 above)
Comment (haszari) [reply includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php]: Probably best to follow that up on P2.

https://woomarketplace.wordpress.com/2019/02/15/extension-discovery-all-screens-i2/

https://woomarketplace.wordpress.com/2019/02/15/extension-discovery-all-screens-i2/#comment-2131

**Comment by haszari** [comment]
Merging into feature branch – all feedback has been addressed, and conflicts resolved.

If I missed something please raise it on feature branch PR! https://github.com/woocommerce/woocommerce/pull/22857

**Pull request merged**